terraform/modules/aws_gundeck-push-notifications/variables.tf

```diff
@@ -0,0 +1,45 @@
+variable "region" {
+  type        = string
+  description = "defines in which region state and lock are being stored"
+  default     = "eu-central-1"
+}
+
+variable "environment" {
+  type        = string
+  description = "name of the environment as a scope for the created resources (e.g., prod. Can be staging or anything else)"
+  default     = "dev"
+}
+
+variable "account_id" {
```

Comment:
Why do we want this to be a parameter rather than using the account ID associated with the caller?

Reply:
When we modularize this logic, I think that assumption is no longer valid.

Reply:
How do u envision that being used? The caller would create resources on different accounts?

Reply:
Thx for bumping my nose into this. It appears, I had sth else in my head, that I compared to a concept in GCP. You are right, it doesnt really make sense.
Though, since we changed `event_delivery_failure_topic_arn` and alike, at least for the gundeck module, we dont need the account ID anymore. That said, since it is still required for gundeck's helm configuration, I suggest, we put the data resource to obtain the account ID somewhere in the TF environment code to output it from there  and be able to set the helm values correctly.